### ISSUE TITLE
Disallow CUD operations

### ISSUE THREAD

**Issue by kako-nawao** [kako-nawao/django-group-by#1]
The instances initialized by the values group query set may contain no ID, meaning that any attempt to _save_ a group will try to create a new object. We should disallow any such operations on both querysets and instances.

**Comment by kako-nawao** [comment]
No, the actual Django instances do contain all the data they need, we don't need to worry about it.

**Comment by nuarhu** [comment]
Hi there,

this really is a helpful module for us. Thank you!

It helped me to create a table with grouped data really quickly. As we are planning to allow modifications on the grouped instances I will try to get the PKs of the grouped instances as an annotation into the dataset somehow. So, CUD on the group itself makes no sense, I completely agree, but changing each of the _actual_ instances in a group would actually be helpful.

Once I come up with a solution for us and it is not too custom I'll keep you posted here.

UPDATE:
With Postgres and Django >= 1.9 it's simple:

```
return MyModel.objects.group_by('name', 'user').annotate(pks=ArrayAgg('id')).distinct()
```

The resulting additional property `pks` is a list of the PKs that have been grouped together.

**Comment by kako-nawao** [comment]
Actually, since grouped fields are turned into instances when they are included as a whole (related field name only), they should be expanded to include all the required information to allow saving the instance properly. I'll run a few tests, but I think this isn't really needed.
